instrumentation/opentelemetry-instrumentation-pymssql/src/opentelemetry/instrumentation/pymssql/package.py

```diff
@@ -0,0 +1,16 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+_instruments = ("pymssql >= 2, < 3",)
```

Comment:
We should probably use the more precise min version that this feature was tested on. I believe it should be `pymssql >= 2.3.2` based off of `test-requirements.txt`

Reply:
I have been using this instrumentation code on a system in production since 2021, and at that time we were using pymssql==2.1.5 (I've validated in our git history). I will put that version as a the minimum requirement.